.github/pull_request_template.md

```diff
@@ -1,14 +1,26 @@
-# Changes
+<!-- Thank you for working on Jamulus and opening a Pull Request! Please fill the following to make the review process straight forward -->
```

Comment:
Please fill **in** (…) **straightforward**

Reply:
Done.

Reply:
It should be "fill **in**" too.